k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
+  labels:
+#The following label will be used for additional node reference, since we go for UID based selection of pod.
```

Comment:
The description is too terse to understand. It introduces UID and then selection of pod via UID which makes it really difficult to udnerstand

Reply:
will change the comment